server/game/gameSystems/MoveCardSystem.ts

```diff
@@ -33,34 +33,37 @@ export class MoveCardSystem<TContext extends AbilityContext = AbilityContext> ex
 
     public eventHandler(event: any, additionalProperties = {}): void {
         const context = event.context;
-        const card = event.card;
         // TODO: remove this completely if determinmed we don't need card snapshots
         // event.cardStateWhenMoved = card.createSnapshot();
         const properties = this.generatePropertiesFromContext(context, additionalProperties) as IMoveCardProperties;
-        if (properties.switch && properties.switchTarget) {
-            const otherCard = properties.switchTarget;
-            card.owner.moveCard(otherCard, card.location);
-        }
-        const player = properties.changePlayer && card.controller.opponent ? card.controller.opponent : card.controller;
-        player.moveCard(card, properties.destination, { bottom: !!properties.bottom });
+        // TODO: Is there a better/cleaner way to handle one or multiple cards here?
+        const cards = [].concat(properties.target);
+        cards.forEach((card) => {
```

Comment:
Do you mind changing this into a proper `for (const card of cards)` loop? Only reason is that when debugging it will completely step over this entire code block if it's in an `<array>.forEach()` call, which is super annoying